tests/data_structures/test_flag_filter.yaml

```diff
@@ -0,0 +1,61 @@
+- name: flag_filter_0x900
+  tags:
+    - miniwdl
+    - flag_filter
+  command: >-
+    miniwdl run -d test-output/. --task validate_string_is_oct_dec_or_hex data_structures/flag_filter.wdl number="0x900"
+
+- name: flag_filter_01
+  tags:
+    - miniwdl
+    - flag_filter
+  command: >-
+    miniwdl run -d test-output/. --task validate_string_is_oct_dec_or_hex data_structures/flag_filter.wdl number="01"
+
+- name: flag_filter_0x1000
+  tags:
+    - miniwdl
+    - flag_filter
+  command: >-
+    miniwdl run -d test-output/. --task validate_string_is_oct_dec_or_hex data_structures/flag_filter.wdl number="0x1000"
+  exit_code: 42
```

Comment:
In the `pytest-workflow` docs, the author uses a `should fail` tag for tasks like this. Should we adopt something similar? Arguably, specifying the non-zero exit code may be sufficient.

Reply:
Yes, I saw that in the `pytest-workflow` docs and thought it was odd. My understanding of the tag feature is to enable/disable groups of tests from the command line. I don't see when you'd ever want to toggle all the "should fail" tests at once.

I think it's safe to skip that tag.

Reply:
I think the use of `tags` is a little overloaded. It could be useful as a documentation field as well as a togglable set of tests to run.